client/src/App.js

```diff
@@ -6,7 +6,7 @@ import {StripeProvider} from 'react-stripe-elements';
 import 'bootstrap/dist/css/bootstrap.min.css'
 import './App.css';
 //PageLoaders
-import Admin from './Components/Admin/Pages/Admin';
+import Admin from './Components/Admin/pages/Admin';
```

Comment:
This bug! Not sure why git keeps lowercasing the directory 'pages' but not the others. As soon as this is merged, there will be compile errors... Just fix by capitalizing this folder name!